config.toml

```diff
@@ -0,0 +1,19 @@
+username = "my-user"
+repository = "https://github.com/my-org/my-repo"
+
+[[labelRule]]
+  [labelRule.create]
+  name = "label1"
+  description = "a cool label"
+  hexColor = "#000000"
+
+[[labelRule]]
+  [labelRule.override]
+  old = "old label name"
+  new = "new label name"
+
+[[ignoredRepository]]
+repo = "https://github.com/my-org/my-ignored-repo"
+
+[[ignoredRepository]]
+repo = "https://github.com/my-org/my-other-ignored-repo"
```

Comment:
Let's actually use simple static arrays here, no need to have table of arrays. Also, the user part is implicitly implied by the `username`, so this array can be just an array of simple repo names, like this:

```toml
ignoredRepositories = [ "my-ignored-repo", "my-other-ignored-repo" ]
```